service/gcs/errors/errors.go

```diff
@@ -1,7 +1,24 @@
-package gcs
+package errors
 
 import (
 	"fmt"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+// Hresult is a type corresponding to the HRESULT error type used on Windows.
+type Hresult int32
+
+const (
+	HrUnexpected   = Hresult(-2147418113) // 0x8000FFFF
```

Comment:
Yeah. And I need to keep it as a signed value, because otherwise it won't marshal correctly when sent back to the HCS (it will marshal as a positive number rather than a negative one).